internal/cache/file/cache_handle_test.go

```diff
@@ -113,16 +115,16 @@ func (cht *cacheHandleTest) SetUp(*TestInfo) {
 	ctx := context.Background()
 	testObjectContent := make([]byte, TestObjectSize)
 	n, err := rand.Read(testObjectContent)
-	AssertEq(TestObjectSize, n)
-	AssertEq(nil, err)
+	assert.Equal(cht.T(), TestObjectSize, n)
+	assert.Equal(cht.T(), nil, err)
 	objects := map[string][]byte{TestObjectName: testObjectContent}
 	err = storageutil.CreateObjects(ctx, cht.bucket, objects)
-	AssertEq(nil, err)
+	assert.Equal(cht.T(), nil, err)
```

Comment:
nit: assertNil
Consider it on other places also.

Reply:
done